scene/main/window.cpp

```diff
@@ -1140,6 +1140,15 @@ void Window::popup(const Rect2i &p_screen_rect) {
 
 	set_transient(true);
 	set_visible(true);
+
+	int screen_id = DisplayServer::get_singleton()->window_get_current_screen(get_window_id());
+	Rect2i screen_rect = Rect2i(DisplayServer::get_singleton()->screen_get_position(screen_id), DisplayServer::get_singleton()->screen_get_size(screen_id));
+	if (!screen_rect.intersects(Rect2i(position, size))) {
```

Comment:
It should check for `screen_rect != Rect2i()`, IIRC some Linux configs unable get correct screen rects so `screen_get_position` and `screen_get_size` might fail.